coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with [c] pattern
```

Comment:
I think `string` is not necessarily a glob expression, the output is. In fact it escapes `string` in such a way that it loses all manners in which it *could* be a glob.